velox/docs/functions/spark/math.rst

```diff
@@ -233,3 +233,11 @@ Mathematical Functions
 .. spark:function:: unaryminus(x) -> [same as x]
 
     Returns the negative of `x`.  Corresponds to Spark's operator ``-``.
+
+.. spark:function:: unhex(x) -> varbinary
+
+    Converts hexadecimal varchar ``x`` to varbinary.
+    If size ``x`` is odd, parses the first character and puts it at index 0 of binary result,
+    then, continues parsing remaining characters, if existing, and
+    still puts the parsing result starting from index 0, which will overwrite the previous value.
+    If ``x`` contains invalid character, return null.
```

Comment:
It would be nice to add some examples. The behavior is very confusing.

Reply:
thanks, already add examples in doc.